storage/doc/design.md

```diff
@@ -420,6 +420,7 @@ but are considered out-of-scope of the current design for time and budget reason
 * external secret-less corruption checks that would use checksums over encrypted data
 * concealing file sizes (though zip chunks helps a bit here)
 * implementing different storage plugins
+* being able to recover data from a partial backup
```

Comment:
this is a bit tricky, because there are no partial backups, so you could say:

* making and persisting incomplete backups